winui/Kanban/column.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Columns in WinUI Kanban control | Syncfusion
+description: Learn here all about configuring and customizing columns in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Column in WinUI Kanban Control
+
+The WinUI Kanban control organizes its layout into columns, representing distinct stages of a workflow or process. Common stages include To-Do, In-Progress, Testing, Validation, and Completed.
+
+## Customizing column width
+
+Columns are smartly sized by default to ensure optimal arrangement and readability of card elements. However, you can customize column widths to suit your specific requirements by using `MinimumColumnWidth` and `MinimumColumnWidth` properties of `SfKanban`.
+
+{% tabs %}
+{% highlight XAML hl_lines="3 4" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 MinimumColumnWidth="300"
+                 MinimumColumnWidth="350">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.MinimumColumnWidth = 300;
+this.kanban.MaximumColumnWidth = 350;
+
+{% endhighlight %}
+{% endtabs %}
+
+You can also define the exact width for the columns using `ColumnWidth` property of `SfKanban`.
+
+{% tabs %}
+{% highlight XAML hl_lines="3" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 ColumnWidth="400">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.ColumnWidth = 400;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Categorizing columns
+
+In the WinUI Kanban Control, cards are categorized and placed into their respective columns based on the value of the `Category` property of `KanbanModel`.
+
+### Populating columns with cards from Multiple categories
+
+You can populate a single column with cards from multiple categories by assigning multiple values to the `Categories` collection of the `KanbanColumn`. For example, to group tasks categorized as `Done` and `Validated` under a single `Done`column, you can include both category values in the `Categories` collection.
```

Comment:
The Kanban board allows to render a single column by mapping multiple categories using Categories property of KanbanColumn in SfKanban. For example, you can map the “In Progress” and “Validated” types under the “In progress” column.

Reply:
Content changed